nest/05.cheat_sheet.md

```diff
@@ -0,0 +1,18 @@
+# 5 Cheat sheet
+In this chapter you will find a cheat sheet, detailing all the commands and options available for `nest` and `finest`, with a brief explanation on what they do, and how to use them.
+
+## `nest`
+* `nest pull` updates the local cache. It must be kept up to date as often as possible 
+* `nest install <package>` installs a package.
+* `nest uninstall <package>` uninstalls a package
+* `nest upgrade <package>` performs a full upgrade of the system, or upgrades the packages provided.
+* `nest list` lists all the installed packages
+
+## `finest`
+* `finest requirement add <req> [--parent <parent>]` adds a requirement to the given parent group (default: `@root`)
+* `finest requirement remove <req> [--parent <parent>]` removes a requirement from the given parent group (default: `@root`)
+* `finest group create <group> [--parent <parent>]` creates a group to the given parent (default: `@root`)
+* `finest group delete <group>` removes a group if the group is not empty, use `-f` or `--force`flag to force the removal
+* `finest group list` lists all existing groups on the OS
+* `finest pull <repository>` pulls all of the repositories if none are given
+* `finest merge` this command must be executed after transactions were executed in order to merge them in your current dependency graph
```

Comment:
This decription doesn't follow the same format than the previous one, which is odd.